addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -416,6 +416,9 @@ export class RelationalModel extends Model {
                         resId: group.value ?? false,
                     };
                 }
+            } else {
+                config.groups[group.value].isFolded =
+                    "__fold" in groupData ? groupData.__fold : config.groups[group.value].isFolded;
```

Comment:
This probably means that if the user manually folded/unfolded groups, those operations will be lost at each reload. I don't think that's what we want.

Reply:
Hello :wave: Thanks for the feedback!
Indeed, did not think about this use case. I removed the commit and created a feedback ticket for more investigating on how we can improve the behavior. Thanks again!